### PR TITLE
Translate API no longer requires an API key.

### PR DESCRIPTION
**Pull request by jmdobry**
See 

- GoogleCloudPlatform/python-docs-samples#659
- GoogleCloudPlatform/ruby-docs-samples#113

### CODE REVIEW

File: translate/quickstart/quickstart.php [hunk 1]
```diff
@@ -7,12 +7,12 @@
 # Imports the Google Cloud client library
 use Google\Cloud\Translate\TranslateClient;
 
-# Your Translate API key
-$apiKey = 'YOUR_API_KEY';
+# Your Google Cloud Platform project ID
+$projectId = 'YOUR_PROJECT_ID';
```
Comment (bshaffer) [inline translate/quickstart/quickstart.php]: Is the project ID required in lieu of an API key?

**Comment by bshaffer** [comment]
Will merge once tests pass

**Comment by jmdobry** [comment]
Can you help me fix the test? I don't understand why it's failing.

**Comment by tmatsuo** [comment]
The linter complains about unused `use` statements

**Comment by bshaffer** [comment]
@jmdobry thanks Jason!

**Pull request merged**